src/main/java/net/dv8tion/jda/api/requests/restaction/MessageAction.java

```diff
@@ -474,4 +493,133 @@ default MessageAction addFile(@Nonnull final File file, @Nonnull AttachmentOptio
+     * <p>
+     * <b>Note:</b> When a User is whitelisted this way, then parsing of User mentions is automatically disabled.
+     * <br>Also note that this has no effect, if the message parsing feature is not enabled (default).
+     *
+     * @param  userIds
+     *         Ids of Users that should be explicitly whitelisted to be pingable.
+     *
+     * @return Updated MessageAction for chaining convenience
+     *
+     * @see #allowedMentions(EnumSet)
+     * @see #setDefaultMentions(EnumSet)
+     */
+    @Nonnull
+    @CheckReturnValue
+    MessageAction mentionUsers(String... userIds);
+
+    /**
+     * Used to provide a whitelist of {@link net.dv8tion.jda.api.entities.User Users} that should be pinged,
+     * even when they would not be pinged otherwise according to the Set of allowed mention types.
+     * <p>
+     * <b>Note:</b> When a User is whitelisted this way, then parsing of User mentions is automatically disabled.
+     * <br>Also note that this has no effect, if the message parsing feature is not enabled (default).
+     *
+     * @param  userIds
+     *         Ids of Users that should be explicitly whitelisted to be pingable.
+     *
+     * @return Updated MessageAction for chaining convenience
+     *
+     * @see #allowedMentions(EnumSet)
+     * @see #setDefaultMentions(EnumSet)
+     */
+    @Nonnull
+    @CheckReturnValue
+    default MessageAction mentionUsers(long... userIds) {
+        String[] stringIds = new String[userIds.length];
+        for (int i = 0; i < userIds.length; i++)
+        {
+            stringIds[i] = Long.toUnsignedString(userIds[i]);
+        }
+        return mentionUsers(stringIds);
+    }
+
+    /**
+     * Used to provide a whitelist of {@link net.dv8tion.jda.api.entities.Role Roles} that should be pinged,
+     * even when they would not be pinged otherwise according to the Set of allowed mention types.
+     * <p>
+     * <b>Note:</b> When a Role is whitelisted this way, then parsing of Role mentions is automatically disabled.
+     * <br>Also note that this has no effect, if the message parsing feature is not enabled (default).
+     *
+     * @param  roleIds
+     *         Ids of Roles that should be explicitly whitelisted to be pingable.
+     *
+     * @return Updated MessageAction for chaining convenience
+     *
+     * @see #allowedMentions(EnumSet)
+     * @see #setDefaultMentions(EnumSet)
+     */
+    @Nonnull
+    @CheckReturnValue
+    MessageAction mentionRoles(String... roleIds);
+
+    /**
+     * Used to provide a whitelist of {@link net.dv8tion.jda.api.entities.Role Roles} that should be pinged,
+     * even when they would not be pinged otherwise according to the Set of allowed mention types.
+     * <p>
+     * <b>Note:</b> When a Role is whitelisted this way, then parsing of Role mentions is automatically disabled.
+     * <br>Also note that this has no effect, if the message parsing feature is not enabled (default).
```

Comment:
This confuses me. What do you mean when you say this doesn't work when the "message parsing feature" is not enabled? Aren't you always able to pass the mentioned roles? From what I can tell the docs say you can do this:

```json
{
  "content": "...",
  "allowed_mentions": {
    "roles": [123, 456]
  }
}
```

Quite the opposite of what you're suggesting here, the API will return an error if both the `parse` and `roles` fields conflict. Am I missing something?